src/Raven.Server/Routing/RequestRouter.cs

```diff
@@ -264,8 +264,19 @@ public static unsafe bool TryGetAccessToken(RavenServer ravenServer, string toke
 
                         var clusterTopology = ravenServer.ServerStore.GetClusterTopology(txContext);
                         var publicKey = clusterTopology.GetPublicKeyFromTag(tag);
+                        //This is the case where we don't know the server but the admin had injected the server's 
+                        //public key into our server store using an external tool.
                         if (publicKey == null)
-                            return false;// we don't know this server, maybe a new one or one that was added?
+                        {
+                            try
+                            {
+                                publicKey = ravenServer.ServerStore.GetSecretKey(txContext, $"Raven/Sign/Public/{tag}");
```

Comment:
this will reside in the topology as @iftahbe implemented it, in the case we can't find it there we will look at the storage and that is injected through an external tool.
it will not throw it will return null, need to change this.